test/Microsoft.AspNetCore.Razor.Evolution.Test/FileSystemRazorProjectTest.cs

```diff
@@ -68,7 +68,7 @@ public void NormalizeAndEnsureValidPath_NormalizesToForwardSlashes()
         public void EnumerateItems_DiscoversAllCshtmlFiles()
         {
             // Arrange
-            var fileSystemProject = new FileSystemRazorProject(TestFolder);
```

Comment:
It's better to call the constructor here. We want to make sure that we're really testing this class.